### PR TITLE
Feature/dx 1221 more consumer and payer fixes

### PR DESCRIPTION
**Pull request by arebra**
I did not touch the modules /SDKs yet, as we are going to rework those and it was a lot of context to unpack.

Changed FROM payer to consumer in a few flows for consistency, so we don't have payer and consumer used interchangeably in the same flowchart.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -17,12 +17,14 @@ GEM
     ethon (0.12.0)
       ffi (>= 1.3.0)
     eventmachine (1.2.7)
-    faraday (1.1.0)
+    eventmachine (1.2.7-x64-mingw32)
```
Comment (NullableInt) [inline Gemfile.lock:20]: Revert these changes please. :)

File: Gemfile.lock [hunk 2]
```diff
@@ -17,12 +17,14 @@ GEM
...
+    faraday (1.0.1)
       multipart-post (>= 1.2, < 3)
       ruby2_keywords
     fastimage (1.8.1)
       addressable (~> 2.3, >= 2.3.5)
     ffi (1.13.1)
+    ffi (1.13.1-x64-mingw32)
```
Comment (NullableInt) [inline Gemfile.lock:27]: Revert these changes please. :)

File: Gemfile.lock [hunk 3]
```diff
@@ -105,6 +107,8 @@ GEM
     multipart-post (2.1.1)
     nokogiri (1.10.10)
       mini_portile2 (~> 2.4.0)
+    nokogiri (1.10.10-x64-mingw32)
+      mini_portile2 (~> 2.4.0)
```
Comment (NullableInt) [inline Gemfile.lock:111]: Revert these changes please. :)

File: Gemfile.lock [hunk 4]
```diff
@@ -115,7 +119,7 @@ GEM
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (4.0.6)
-    rack (2.1.4)
+    rack (1.6.13)
```
Comment (NullableInt) [inline Gemfile.lock:122]: Revert these changes please. :)

File: Gemfile.lock [hunk 5]
```diff
@@ -133,6 +137,8 @@ GEM
       rb-inotify (~> 0.9, >= 0.9.7)
     sassc (2.4.0)
       ffi (~> 1.9)
+    sassc (2.4.0-x64-mingw32)
+      ffi (~> 1.9)
```
Comment (NullableInt) [inline Gemfile.lock:141]: Revert these changes please. :)

File: Gemfile.lock [hunk 6]
```diff
@@ -173,6 +179,7 @@ GEM
 
 PLATFORMS
   ruby
+  x64-mingw32
```
Comment (NullableInt) [inline Gemfile.lock:182]: Revert these changes please. :)

**Review by NullableInt — CHANGES_REQUESTED** [review]
Please revert the changes you've done to `Gemfile.lock` 🥰

**Review by asbjornu — APPROVED** [review]

**Review by NullableInt — APPROVED** [review]
